### ISSUE TITLE
Curious how you train for 160 frames, which seems to need 160x128 tokens?

### ISSUE THREAD

**Issue by leexinhao** [KangarooGroup/Kangaroo#4]
I noticed that you used up to 160 frames for training, how did you do that, did you use some acceleration and parallelism outside of the algorithm like [LongVILA](https://www.arxiv.org/abs/2408.10188)?

**Comment by IceWYB** [comment]
Sure, we use pipeline parallelism (PP=4) and gradient checkpointing for instruction tuning.

**Comment by leexinhao** [comment]
> Sure, we use pipeline parallelism (PP=4) and gradient checkpointing for instruction tuning.

Thank you very much for your excellent work. In addition, I am curious about the impact of long video tuning on the final performance. For example, how much can be improved in VideoMME and mlvu? I see that this point is not mentioned in the paper.

**Comment by IceWYB** [comment]
According to our preliminary test, using more frames for short videos provides limited benefits. We are still working on further improving the results on long video benchmarks and will report its impact in the upcoming versions.
